packages/bsky/src/api/app/bsky/feed/getFeed.ts

```diff
@@ -137,3 +219,26 @@ async function skeletonFromFeedGen(
     feed: skeleton.feed.slice(0, params.limit), // enforce limit
   }
 }
+
+const cleanFeedSkeleton = async (
```

Comment:
this isn't really "cleaning" anymore right? it's more of a "pre-hydration hydration" step 🤔 

mainly just the naming thing

Reply:
but also if we're feeling adventurous, seems that we could cut out some duplicate work if we moved this to the skeleton step & only did it for 3rd party feeds

Reply:
Done!